### PR TITLE
Alert and badge

### PR DESCRIPTION
**Pull request by purvasharmadev**
Added two components Alert and Badge.

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Code is clean and no issues with naming conventions. file structure is also good. Comments are descriptive. all good to go

File: component.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    --SECONDARY-COLOR:#a855f7;
+    --BACKGROUND-COLOR:YELLOW;
+    --TEXT-COLOR:#e9d5ff;
+
+    /* Component Colors */
+    --DANGER-COLOR:#dc2626;
+    --SUCESS-COLOR:#65a30d;
+    --INFO-COLOR:#2563eb;
+    --WARNING-COLOR:#ca8a04;
+
```
Comment (varun13169) [inline component.css]: Nit: Space

**Review by varun13169 — CHANGES_REQUESTED** [review]

**Review by dcs-soni — APPROVED** [review]
Looks good. Well structured 👍

**Comment by ghost** [comment]
Well Formatted and Structured Code 👍

**Review by ghost — APPROVED** [review]

File: component.css (hunk 1 above)
Comment (purvasharmadev) [reply component.css]: Reduced the changes. Thankyou for reviewing.

**Comment by purvasharmadev** [comment]
> Code is clean and no issues with naming conventions. file structure is also good. Comments are descriptive. all good to go

Thankyou for your Valuable Review.

**Pull request merged**